README.md

```diff
@@ -1 +1,20 @@
-# eq-runner-docker-images
\ No newline at end of file
+# eQ Runner Docker Images
+
+This repository contains Dockerfiles used to build base images for:
+- Deploying eQ container with helm  ([app-deploy-image](k8s-helm-deploy-image))
+- Building the eQ infrastructure base image 
+- Building the Tinyproxy image used for eq-runner-concourse
+- Building the eQ submission confirmation consumer deploy image
+
+The first three images are used in Concourse pipelines to run the tasks above and the final image is used to connect to Concourse Kubernetes.
+### Building the images
```

Comment:
```suggestion
The first three images are used in Concourse pipelines to run the tasks above and the final image is used to connect to Concourse Kubernetes.

## Building the images
```

Would be good to have a space here between the heading and the text above and this heading should be a level 2 title to fit with the level 1 one you have at the top

Reply:
Updated 👍